### PR TITLE
DOC: reasoning for communications guidelines

### PR DESCRIPTION
**Pull request by story645**
## PR summary

Follow up to #26703, last winter(ish) @melissawm, @noatamir, and me worked on writing up the reasoning for the communications guidelines and objectives for the strategy. I figure especially w/ the move to the main repo, it's worth having it in the doc as grounding for the guidelines. 

## PR checklist

- [x] Documentation complies with [general](https://matplotlib.org/devdocs/devel/document.html#write-rest-pages) and [docstring](https://matplotlib.org/devdocs/devel/document.html#write-docstrings) guidelines

### CODE REVIEW

File: doc/devel/communication_guide.rst [hunk 1]
```diff
@@ -4,11 +4,32 @@
...
+The Scientific Python community uses various communications platforms to stay
+updated on new features and projects, to contribute by telling us what is on
+their mind and suggest issues and bugs, and to showcase their use cases and the
+tools they have built.
+
+Our communications strategy is foremost guided by our :ref:`mission-statement`:
+
+* We demonstrate that we care about visualization as a practice
+* We deepen our practice and the community’s capacity to support users,
+  facilitate exploration, produce high quality viz, and be understandable and
```
Comment (timhoffm) [inline doc/devel/communication_guide.rst:16]: Is _vis_ slang?

File: doc/devel/communication_guide.rst [hunk 2]
```diff
@@ -4,11 +4,32 @@
...
+  extensible
+* We showcase advanced use of the library without adding maintenance burden to
+  the documentation and show recognition for work that may not be in the
+  “what’s new” section.
```
Comment (timhoffm) [inline doc/devel/communication_guide.rst:20]: What kind of work is this? Can you write that positive, i.e. not as a negation?

File: doc/devel/communication_guide.rst [hunk 3]
```diff
@@ -4,11 +4,32 @@
...
+* We use communications platforms to maintain relationships with contributors
+  who may no longer be active on GitHub, build relationships with potential
+  contributors, and connect with other projects and communities who use
+  Matplotlib.
+* In prioritizing understandability and extensiblity, we empower people working
+  with Matplotlib to feel like they are a part of our community and that they are
+  doing valuable community building labor.
```
Comment (timhoffm) [inline doc/devel/communication_guide.rst:27]: Is "working with Matplotlib" already valueable community building? Maybe "building on top of Matplotlib"? Also, you can leave out "labor", "doing valuable community building." is enough.

File: doc/devel/communication_guide.rst [hunk 4]
```diff
@@ -148,12 +169,32 @@ Visual media in communications should be made as accessible as possible:
...
 Social media
 ============
 
-Please follow these guidelines to maintain a consistent brand identity across
-platforms.
+Matplotlib aims for a single voice across all social media platforms to build and
+maintain a consistent brand identity for Matplotlib as an organization. This
+depersonalization is the norm on social media platforms because it enables
+constructive and productive conversations; People generally feel more comfortable
+giving negative and constructive feedback to a brand then to specific contributors.
```
Comment (timhoffm) [inline doc/devel/communication_guide.rst:181]:
```suggestion
giving negative and constructive feedback to a brand than to specific contributors.
```

**Comment by timhoffm** [comment]
Semi-OT: When I saw this under "Contribute > Communication Guidelines" I got the impression that people wanting to contribute should read this and that it relates to how we communicate on github for issues and PRs.

Should we rename the page to "_Public_ communication guidelines" (or "_Social media_ communication guidelines", which is maybe slightly too narrow but even more clear that not every contributor does not have to care about it.

**Comment by timhoffm** [comment]
The comm guide state the scope at the top

> These guidelines are applicable when acting **as a representative of Matplotlib**, for example at sprints or when giving official talks or tutorials, and in any community venue managed by Matplotlib.

I'd like to capture this in the title, in particular with the intent to make it obvious for a first-time contributor that they do not need to worry about this when posting an issue or PR.

**Comment by timhoffm** [comment]
> What in the guide do you think would make a new contributor think that they DO have to worry about this?

Nothing *in* there, it's the title that is too generic. When you go to contribution entry page https://matplotlib.org/devdocs/devel/index.html, you get a side bar with lots of entries:

image

which is quite overwhelming, For some, like *Release guide*, it's clear that I as a first-time contributor does not have to care about. It would be helpful if that would come across also in the communication heading (and/or possibly better separation between topics relevant to the majority of contributors, and topics only relevant to special people).

**Comment by jklymak** [comment]
The organization of the index is ideally the organization of the cards.

**Review by timhoffm — APPROVED** [review]
Nice and inviting description! 🥇

**Pull request merged**